ajax/libs/murmurhash3js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "murmurhash3js",
+  "filename": "murmurhash3js.min.js",
+  "version": "3.0.1",
+  "homepage": "https://pid.github.io/murmurHash3js/",
+  "description": "A javascript implementation of MurmurHash3's x86 hashing algorithms.(Browser&Server)",
+  "keywords": [
+    "murmurhash3",
+    "hash"
+  ],
+  "author": {
+    "name": "Karan Lyons",
+    "email": "karan@karanlyons.com",
+    "url": "https://karanlyons.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pid/murmurHash3js.git"
+  },
+  "npmName": "murmurhash3js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "murmurhash3js.min.js"
+      ]
+    },
+    {
+      "basePath": "lib",
+      "files": [
+        "murmurHash3js.js"
```

Comment:
In this case, we can just use `murmurHash3js*`. If there are some files we don't need mixed in the same directory, we need to use more restricted rule.